packages/aws-cdk/test/api/hotswap/hotswap-test-setup.ts

```diff
@@ -81,9 +81,17 @@ export class HotswapMockSdkProvider {
     });
   }
 
-  public setUpdateFunctionCodeMock(mockUpdateLambdaCode: (input: lambda.UpdateFunctionCodeRequest) => lambda.FunctionConfiguration) {
+  public setUpdateFunctionCodeMock(
```

Comment:
The name of this method doesn't really fit anymore.

Let's just call this `stubLambda()` instead, and surface the same API `MockSdkProvider` does (it's nicer, because the object simulates named parameters).